src/Types/Interface/Channel.ts

```diff
@@ -0,0 +1,9 @@
+export interface Channel {
+  authRequired: boolean; // 사용되지 않음
```

Comment:
사용되지 않는 모델 프로퍼티는 일단 선택속성으로 해두는게 좋을 듯 해요!
```suggestion
  authRequired?: boolean; // 사용되지 않음
```